R/getClusterAnalysis.R

```diff
@@ -142,12 +143,12 @@ getClusterAnalysis = function(data, num.features, method, par.vals, random.seed,
     } else if (method == "cluster.diana") {
       cluster.method = "diana"
     }
-    out.clust = do.call(eclust, args = append(list(x = num.data, FUNcluster = cluster.method, verbose = FALSE, k = 4), par.vals))
```

Comment:
@ptl93 are you fine with this change?

Reply:
Hey, yeah sure. I forgot to change that. Indeed, you are right to let it flexible and not determine the number of groups for the hierarchical clustering algorithms. I am fine with the chane. Thx

Reply:
I merged my branch and caused a few merge conflicts in getClusterAnalysis. Would be good if you can add your version of number of groups within the hierarchical methods.